src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import Page from './Page';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Page from './Page';
```
공백 추가해주세요~ 모듈 간에도 역할이 있고, 지금은 import 하는게 별로 없어 괜찮을 수 있지만, 묶어서 관리하는 습관을 들이면 좋을 것 같습니다~

다른 파일들도 확인해보시고 적용해주세요~ 👍

Reply:
@Kihyun92 모듈 종류를 '컴포넌트'와 '라이브러리'로 구분하기 위함인가요?

Reply:
정확합니다~ 지금은 라이브러리와 컴포넌트 밖에 없지만 프로젝트가 커질수록 사용되는 모듈의 종류도 매우 많아지게 되는데요. 이때 잘 묶어두면 훨씬 보기 쉽고 수정하기 쉽겠죠?

lint중에 `ordered-imports`를 사용해서 파일 형식에 따라 순서를 잡아주는게 있는데 이걸 쓰기도 합니다 :)

Reply:
그렇군요 그 rule 도 켜놓고 해야겠네요 ㅎㅎ
좋은것 알려주셔서 감사합니다!